ocaml/xapi-idl/network/network_interface.ml

```diff
@@ -34,19 +34,26 @@ let comp f g x = f (g x)
 let ( ++ ) f g x = comp f g x
 
 let netmask_to_prefixlen netmask =
-  Scanf.sscanf netmask "%d.%d.%d.%d" (fun a b c d ->
-      let rec length l x = if x > 0 then length (succ l) (x lsr 1) else l in
-      let masks = List.map (( - ) 255) [a; b; c; d] in
-      32 - List.fold_left length 0 masks
+  let process_ipaddr_err = function
```

Comment:
Would use `raise`, `fail` or similar in the name to indicate that we are reporting a failure and not merely processing something.